### PR TITLE
Bugfix/working simple examples

### PR DESCRIPTION
**Pull request by claweyenuk**
- [x] Any changes relevant to users are recorded in the `CHANGELOG.md`.
- [x] The documentation has been updated, if necessary.
---

I don't think this is mergeable, but I don't have the time to clean this up at the moment, particularly the keyjar code.  But I thought someone might find this useful since there are issues like #212 and it is mentioned in the main README.

The steps in the README in here took some time to figure out as well, so hopefully that can help some people.

Note that I tested this using Python 3.  I did run this under Python 2 and it required a few very bad hacks to get working in the wsgi lib that I think are because start response may have been called twice.  I did not dig into that

### CODE REVIEW

**Comment by tpazderka** [comment]
Since the examples are SO broken, I am not against merging this with some stuff unfinished. Can you please update this to a current master as some stuff got already fixed?

**Comment by claweyenuk** [comment]
I think this one is good (enough) to go now

**Comment by claweyenuk** [comment]
The example README calls out #543 which I did not know how to work around

File: oidc_example/simple_rp/requirements.txt [hunk 1]
```diff
@@ -1,4 +1,3 @@
-oic==0.7.6
 pyjwkest==1.0.1
```
Comment (tpazderka) [inline oidc_example/simple_rp/requirements.txt]: This should be included in the installation and is probably extra.

File: oidc_example/simple_op/requirements.txt [hunk 2]
```diff
@@ -1,6 +1,5 @@
 cherrypy==3.2.4
 pyaml==15.03.1
 Jinja2==2.7.3
-oic==0.7.6
 yubico-client==1.9.1
 pyjwkest==1.0.1
```
Comment (tpazderka) [inline oidc_example/simple_op/requirements.txt]: This should be included in the installation and is probably extra.

File: oidc_example/simple_rp/requirements.txt (hunk 1 above)
Comment (claweyenuk) [reply oidc_example/simple_rp/requirements.txt]: Should this be worded "should not be included"?

Comment (tpazderka) [reply oidc_example/simple_rp/requirements.txt]: Yes, sorry. Applies to both comments.

**Comment by tpazderka** [comment]
Merged manually.

**Pull request closed without merging**